packages/block-editor/src/components/text-alignment-control/stories/index.story.js

```diff
@@ -8,32 +8,84 @@ import { useState } from '@wordpress/element';
  */
 import TextAlignmentControl from '../';
 
-export default {
+/**
+ * Text alignment options
+ */
+const ALIGNMENT_OPTIONS = [ 'left', 'center', 'right', 'justify' ];
+
+/**
+ * TextAlignmentControl Properties
+ */
+const meta = {
 	title: 'BlockEditor/TextAlignmentControl',
 	component: TextAlignmentControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component: 'Control to facilitate text alignment selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
-		className: { control: 'text' },
+		value: {
+			control: { type: null },
+			description: 'Currently selected text alignment value.',
+			table: {
+				type: {
+					summary: 'string',
+				},
+			},
+		},
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
+			description: 'Handles change in text alignment selection.',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+		},
 		options: {
 			control: 'check',
-			options: [ 'left', 'center', 'right', 'justify' ],
+			description: 'Array of text alignment options to display.',
+			options: ALIGNMENT_OPTIONS,
```

Comment:
```suggestion
			options: [ 'left', 'center', 'right', 'justify' ],
```

The `ALIGNMENT_OPTIONS` variable will only be used once, so let's hard-code it here.